cumulus/primitives/core/src/lib.rs

```diff
@@ -461,6 +467,44 @@ pub struct CollationInfo {
 	pub head_data: HeadData,
 }
 
+/// The schedule for the next relay chain slot.
+///
+/// Returns the number of parachain blocks to produce and the block time per block to use.
+#[derive(Clone, Debug, codec::Decode, codec::Encode, PartialEq, TypeInfo)]
+pub struct NextSlotSchedule {
+	/// The number of blocks to produce in the relay chain slot.
+	pub number_of_blocks: u32,
+	/// The target block time in wall clock time for each block.
+	///
+	/// The maximum should be [`REF_TIME_PER_CORE_IN_SECS`] or otherwise blocks may fail to
```

Comment:
The comment is wrong or I don't understand how this works. I'd expect the value here is not related to how much ref time a core provides.

If I have 2 cores, the block time should be 3s.

Reply:
Block time != block interval. The block time is the time per block for its production.